tests/queries/0_stateless/02346_into_outfile_and_stdout.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+function perform()
+{
+    local test_id=$1
+    local query=$2
+
+    echo "performing test: $test_id"
+    ${CLICKHOUSE_CLIENT} --query "$query" 2>/dev/null
```

Comment:
Tests should not swallow errors unless it's necessary.

Reply:
Thanks. this has been fixed